spot-electron/config.js

```diff
@@ -2,6 +2,17 @@ const process = require('process');
 
 module.exports = {
 
+    /**
+     * Config values for the beacon functionality.
+     */
+    beacon: {
+
+        /**
+         * Region ID for the beacon. This is permanent and must be unique for custom deployments.
+         */
+        region: 'bf23c311-24ae-414b-b153-cf097836947f'
```

Comment:
Do you know what the plan might be for deployments to use? A backend will provide the beacon and clients will do some exchange with the backend?

Reply:
Good question. This has to be fixed in order to allow the iOS app to detect it even when the app is not running. Yes, I think it should come from the deployment later. Is there already a way to do that or we need to wait for the backend?

Reply:
There isn't a way I know of and I'm generally unfamiliar with the work/discussion done for bluetooth before you started working on it. (I saw the demo a while and that might be the sum of my knowledge.)

Reply:
Ok, let's leave it here now, and we adapt when we finally have backend